doc/cephfs/cephfs-mirroring.rst

```diff
@@ -5,7 +5,7 @@ CephFS Snapshot Mirroring
 =========================
 
 CephFS supports asynchronous replication of snapshots to a remote CephFS file system via
-the `cephfs-mirror` tool. Snapshots are synchronized by mirroring snapshot data followed by
+the *cephfs-mirror* tool. Snapshots are synchronized by mirroring snapshot data followed by
```

Comment:
When a name is a literal, e.g. the name of the command or the daemon process, I usually prefer double backticks so it is rendered in monospace, vs single stars which render as italic.

Reply:
I was thinking that this might refer to the system as a whole instead of the specific binary (that's referred to as "`cephfs-mirror` daemon" in all subsequent text). Thanks, will update this.